source/connect-atlas-cli.txt

```diff
@@ -96,6 +96,16 @@ Complete the Prerequisites
 
       - :ref:`Install the {+atlas-cli+} <install-atlas-cli>`.
       - Add your host's IP address to the :ref:`IP access list <access-list>`.
+  
+        If you log into the {+atlas-cli+} with your Atlas user credentials,
```

Comment:
Nit: they don't log in to the CLI, they log in to Atlas using the CLI. Also, no comma needed here
```suggestion
        If you authenticate with your |service| user credentials
```